src/components/common/YourCompany/YourCompany.vue

```diff
@@ -466,7 +467,10 @@ export default class YourCompany extends Mixins(
       return this.getResource.changeData.nameChangeOptions
         .filter(option => option !== CorrectionTypes.CORRECT_NAME_TO_NUMBER)
     }
-    if (this.isEntityTypeFirm) return Array(CorrectionTypes.CORRECT_NEW_NR)
+    if (this.isEntityTypeFirm) {
```

Comment:
No. We're trying to keep components generic (ie, to know as little as possible about the different variations depending on entity type).

As I have mentioned before, there are resource files that specify the name change options for each entity type.

Reply:
Looks good.